packages/styled-components/src/models/StyleSheetManager.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useContext, useEffect, useMemo, useState } from 'react';
 import shallowequal from 'shallowequal';
+import type stylis from 'stylis';
```

Comment:
I'm also open to switching this to `import * as stylis from 'stylis';` to match the other imports in the codebase (it seems type-only imports aren't used much elsewhere?)